File: Java/src/main/java/com/gildedrose/categories/Sulfuras.java
package com.gildedrose.categories;

import com.gildedrose.*;

public class Sulfuras implements ItemCategory {

    public String getName() {
        return "Sulfuras, Hand of Ragnaros";
    }

    public void UpdateQuality(Item item) {
        item.quality = 80;
    }

    public void UpdateSellIn(Item item) {
        // do nothing
    }
}

File: csharpcore/Categories/Sulfuras.cs
namespace csharpcore.Categories
{
    public class Sulfuras : IItemCategory
    {
        public string Name => "Sulfuras, Hand of Ragnaros";

        public void UpdateQuality(Item item)
        {
            item.Quality = 80;
        }

        public void UpdateSellIn(Item item)
        {
            // do nothing
        }
    }
}